mllib/src/test/java/org/apache/spark/ml/regression/JavaRandomForestRegressorSuite.java

```diff
@@ -80,6 +80,14 @@ public void runDT() {
     for (String featureSubsetStrategy: RandomForestRegressor.supportedFeatureSubsetStrategies()) {
       rf.setFeatureSubsetStrategy(featureSubsetStrategy);
     }
+    String realStrategies[] = {".1", ".10", "0.10", "0.1", "0.9", "1.0"};
+    for (String strategy: realStrategies) {
+      rf.setFeatureSubsetStrategy(strategy);
+    }
+    String integerStrategies[] = {"1", "10", "100", "1000", "10000"};
```

Comment:
Also, what happens with negative values? Those should not be allowed - just want to confirm the regex excludes that (we should add some test cases)